src/framework/pagination/_pagination.scss

```diff
@@ -0,0 +1,46 @@
+.pagination {
+  line-height: 1;
+  margin: 1em auto;
+  text-align: center;
+}
+.pagination ul {
+  -moz-user-select: none;
+  -khtml-user-select: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
```

Comment:
I think we have mixins, we probably don't specify vendor prefixes manually here.